src/renderers/webgl/WebGLProgram.js

```diff
@@ -491,6 +491,8 @@ THREE.WebGLProgram = ( function () {
 				parameters.shadowMapEnabled ? '#define ' + shadowMapTypeDefine : '',
 				parameters.pointLightShadows > 0 ? '#define POINT_LIGHT_SHADOWS' : '',
 
+				parameters.premultipliedAlpha ? "#define PREMULTIPLIED_ALPHA" : '',
```

Comment:
We need to modify `WebGLPrograms.js` too so `premultipliedAlpha` is added in `parameters`.